evm_loader/program/src/instruction/mod.rs

```diff
@@ -204,6 +254,10 @@ impl EvmInstruction {
             0x35 => Self::TransactionStepFromAccount,        // 53
             0x36 => Self::TransactionStepFromAccountNoChainId, // 54
             0x37 => Self::Cancel,                            // 55
+            0x38 => Self::TransactionExecuteFromInstructionWithSolanaCall, // 56
+            0x39 => Self::TransactionExecuteFromAccountWithSolanaCall, // 57
+            0x3A => Self::TransactionStepFromAccountSolanaCall, // 58
```

Comment:
Are you going to delete 0x3A => Self::TransactionStepFromAccountSolanaCall ? Or it should work in future?

Reply:
Removed